ts/components/session/usingClosedConversationDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import { GroupUtils } from '../../session/utils';
+import { UserUtil } from '../../util';
+import { PubKey } from '../../session/types';
+import React from 'react';
+import { ConversationAttributes } from '../../../js/models/conversations';
+type State = {
+  closedMemberConversations?: Array<ConversationAttributes>;
+};
+
+export function usingClosedConversationDetails(WrappedComponent: any) {
+  return class extends React.Component<any, State> {
+    constructor(props: any) {
+      super(props);
+      this.state = {
+        closedMemberConversations: undefined,
+      };
+    }
+
+    public componentDidMount() {
+      void this.fetchClosedConversationDetails();
+    }
+
+    public render() {
+      return (
+        <WrappedComponent
+          closedMemberConversations={this.state.closedMemberConversations}
+          {...this.props}
+        />
+      );
+    }
+
+    private async fetchClosedConversationDetails() {
```

Comment:
will there be a case where we need to re-fetch these details? 🤔

Reply:
yes, I am making a change to refresh it